docs/kb/frontend.md

```diff
@@ -7,11 +7,11 @@ order: 8
 
 A Vue frontend app is created in the `vue` directory when a blockchain is scaffolded. To start the frontend app run `npm i && npm run serve` in the `vue` directory.
 
-The frontend app is built using the `@ignt/vue` and `@ignt/vuex` packages. For details, see the [monorepo for Ignite CLI front-end development](https://github.com/ignite-hq/vue).
+The frontend app is built using the `@ignt/vue` and `@ignt/vuex` packages. For details, see the [monorepo for Ignite CLI front-end development](https://github.com/ignite-hq/web).
```

Comment:
@ilgooz what are the package names?
`@ignt/vue` and `@ignt/vuex` packages ??

Reply:
@marinhoarthur